ihatemoney/utils.py

```diff
@@ -1,5 +1,7 @@
 import base64
 import re
+import ast
+import operator
```

Comment:
Adding `from __future__ import division` would make it possible to return floats from integer divisions (in other words, without it `3/2` returns `1`, and with it it returns `1.5`. I believe we should also add a test for this.

Reply:
Good Call! Added import as well as test.